src/sparsezoo/model/model.py

```diff
@@ -78,12 +79,14 @@ def __init__(self, source: str, download_path: Optional[str] = None):
 
         if self.source.startswith(ZOO_STUB_PREFIX):
             # initializing the files and params from the stub
-            files, path, url = self.initialize_model_from_stub(self.source)
+            _model_args = self.initialize_model_from_stub(self.source)
+            files, path, url, validation_results, compressed_size = _model_args
             if download_path is not None:
                 path = download_path  # overwrite cache path with user input
         else:
             # initializing the model from the path
             files, path, url = self.initialize_model_from_directory(self.source)
+            validation_results = compressed_size = None
```

Comment:
Is there a reason why we can't load validation results and compressed size from a downloaded directory? Compressed size seems unlikely since the compressed files are often not downloaded, but however the SparseZoo api parses validation results should be the same method as getting them from a directory